### PR TITLE
Add IPO 1.16.0 easyconfig

### PR DESCRIPTION
**Pull request by rpep**
For INC1129346 - `IPO-1.16.0-foss-2020b.eb`

* [X] Assigned to reviewers (usually everyone in apps team)

Default:
* [ ] EL8-cascadelake
* [ ] EL8-haswell

### CODE REVIEW

File: easybuild/easyconfigs/i/IPO/IPO-1.16.0-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz'
+}
+
+
+exts_defaultclass = 'RPackage'
+
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+exts_list = [
```
Comment (edmondac) [inline easybuild/easyconfigs/i/IPO/IPO-1.16.0-foss-2020b.eb:32]:
```suggestion
exts_list = [
    ('rsm', '2.10.2', {
        'checksums': ['66446cfb4298f1b306be286d1f2cd8b7f68acc900d3cc0d9a054aa48e192091c'],
    }),
```

File: easybuild/easyconfigs/r/rsm/rsm-2.10.2-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+easyblock = 'Bundle'
```
Comment (edmondac) [inline easybuild/easyconfigs/r/rsm/rsm-2.10.2-foss-2020b.eb:1]: You can delete this file (and include it as an ext in IPO, as per my suggested edit above)

**Review by edmondac — CHANGES_REQUESTED** [review]
Please merge together as per my suggestion

File: easybuild/easyconfigs/i/IPO/IPO-1.16.0-foss-2020b.eb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+homepage = 'https://bioconductor.org/packages/release/bioc/html/IPO.html'
+description = """Automated Optimization of XCMS Data Processing parameters"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('R', '4.0.3'),
+    ('R-bundle-Bioconductor', '3.12', '-R-%(rver)s'),
+    ('rsm', '2.10.2'),
```
Comment (edmondac) [inline easybuild/easyconfigs/i/IPO/IPO-1.16.0-foss-2020b.eb:14]:
```suggestion
```

**Review by branfosj — APPROVED** [review]

**Review by edmondac — APPROVED** [review]
lgtm

**Pull request merged**